### PR TITLE
Add listing for Mopidy-YTMusic

### PR DESCRIPTION
**Pull request by impliedchaos**
Hi, just wanting to add Mopidy-YTMusic backend to the directory.

Thanks!

### CODE REVIEW

**Comment by kingosticks** [comment]
I think it would be more useful if there was a comparison between this and Mopidy-YouTube. Why are there two? Why would I want one or the other?

**Comment by impliedchaos** [comment]
I think it's in the titles.  Mopidy-Youtube is for listening to audio from youtube videos.  Mopidy-YTMusic is for listening to songs from YouTube Music (https://music.youtube.com/).

**Comment by kingosticks** [comment]
I would look at the titles and assume Mopidy-Youtube did it all. But maybe that is because I am unfamiliar with this month's flavour of music offering from Google. You are welcome to submit a PR making the distinction clearer on either page if you change your mind.

File: _ext/ytmusic.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: Mopidy-YTMusic
+service: YouTube Music
+type: backend
+dev:
+  github: OzymandiasTheGreat/mopidy-ytmusic
+dist:
+  pypi: Mopidy-YTMusic
+logo: /media/ext/youtubemusic.svg
+images:
+- /media/ext/youtubemusic.svg
```
Comment (kingosticks) [inline _ext/ytmusic.md:11]:
```suggestion
```
`images` is for images showing the extension in use.

Comment (impliedchaos) [reply _ext/ytmusic.md:11]: Thanks, I was clearly not paying close enough attention.

**Comment by impliedchaos** [comment]
> I would look at the titles and assume Mopidy-Youtube did it all.

Yeah, I'm OK with that assumption and I think you're right. It's Google's fault that their services are confusingly named and unknown, but I'm not here to help them sell subscriptions.  I just want to offer those that do know the option to listen from mopidy.  So I'm not sure how much clearer I can make it than by listing the name (`service: YouTube Music`) and logo.   Maybe `service: The rubbish Google left us with after cancelling Google Play Music.`?

**Comment by kingosticks** [comment]
Thanks for adding and for your work on the extension.

**Comment by impliedchaos** [comment]
Thank you!

**Comment by kingosticks** [comment]
Ah, but the logo doesn't seem to be working for me.

**Comment by impliedchaos** [comment]
Hmm... It does for me:
image

Let me try rasterizing to png and updating it with that instead.

**Comment by kingosticks** [comment]
It's weird, it's obviously available at the URL and I can open just the image but it won't display on the page (Firefox).

**Pull request merged**